coala_quickstart/generation/FileGlobs.py

```diff
@@ -7,7 +7,7 @@
 from coalib.collecting.Collectors import collect_files
 
 
-def get_project_files(log_printer, printer, project_dir):
+def get_project_files(log_printer, printer, project_dir, noninteractive):
```

Comment:
This is a little unclear to me... I mean these tests :(. Why to make non_interactive False here, in FileGlobs? I receive it in the definition of the function. Correct me if I'm wrong...